modules/replication/dockerimage-replication/deployspec.yaml

```diff
@@ -8,15 +8,19 @@ deploy:
         - mv linux-amd64/helm /usr/local/bin/helm
     build:
       commands:
+        - |
+          if ! aws s3api head-bucket --bucket "addf-metadata-${AWS_DEFAULT_REGION}-${AWS_ACCOUNT_ID}"; then
+            aws s3api create-bucket --bucket "addf-metadata-${AWS_DEFAULT_REGION}-${AWS_ACCOUNT_ID}" --region "${AWS_DEFAULT_REGION}" --create-bucket-configuration LocationConstraint=${AWS_DEFAULT_REGION}
+          fi
         - |
           if [[ -n ${ADDF_PARAMETER_NO_REPLICATE} ]]; then
-            python3 get-list-of-eks-images.py --eks-version ${ADDF_PARAMETER_EKS_VERSION} --versions-directory versions --update-helm-repos --registry-prefix "${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/${AWS_CODESEEDER_NAME}-" --no-replicate
+            python3 get-list-of-eks-images.py --eks-version ${ADDF_PARAMETER_EKS_VERSION} --versions-directory data/eks_dockerimage-replication/versions --update-helm-repos --registry-prefix "${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/${AWS_CODESEEDER_NAME}-" --no-replicate
           else
-            python3 get-list-of-eks-images.py --eks-version ${ADDF_PARAMETER_EKS_VERSION} --versions-directory versions --update-helm-repos --registry-prefix "${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/${AWS_CODESEEDER_NAME}-"
+            python3 get-list-of-eks-images.py --eks-version ${ADDF_PARAMETER_EKS_VERSION} --versions-directory data/eks_dockerimage-replication/versions --update-helm-repos --registry-prefix "${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/${AWS_CODESEEDER_NAME}-"
           fi
         - chmod +x replication.sh
         - bash replication.sh create
-        - export ADDF_MODULE_METADATA=$(python -c "import yaml; import json; file=open('images.yaml'); print(json.dumps(yaml.safe_load(file)))")
```

Comment:
export metadata at the end and write the s3 file+key location so you can consume it downstream?